pkg/logs/tekton_logging_test.go

```diff
@@ -112,6 +112,39 @@ func TestGetTektonPipelinesWithActivePipelineActivityOnlyWaitingStep(t *testing.
 	assert.Equal(t, 1, len(paNames))
 }
 
+// Based on a real case
+func TestGetTektonPipelinesWithActivePipelineActivityInvalidName(t *testing.T) {
+	testCaseDir := path.Join("test_data", "invalid_name")
```

Comment:
ditto

Reply:
If you meant that the directory wasn't cleaned up, there is not file or directory created in that test.

Reply:
Yeah. I see. Thanks for follow up.